modules/gapi/src/backends/cpu/gcpucore.cpp

```diff
@@ -11,6 +11,112 @@
 #include <opencv2/gapi/cpu/core.hpp>
 #include <opencv2/gapi/cpu/gcpukernel.hpp>
 
+class YoloParser
```

Comment:
I suggest to look at [gapi_video_tests_common.hpp](https://github.com/opencv/opencv/blob/master/modules/gapi/test/common/gapi_video_tests_common.hpp#L58) where structures/classes have conventions/rules proposed to avoid compiler/KW warnings

all the fields with types like `int` or `float` should be pre-initialized; and if a structure has a constructor, sometimes to avoid warnings there should be also a [default constructor](https://github.com/opencv/opencv/blob/master/modules/gapi/test/common/gapi_video_tests_common.hpp#L83) (I hope it may be helpful)

Reply:
I put functions in class in the tests and create a class for parseSSD (no more warnings).